src/renderer/modals/UpdateFirmwareModal/steps/01-step-install-full-firmware.js

```diff
@@ -90,7 +90,10 @@ const Body = ({
           {t("manager.modal.identifier")}
         </Text>
         <Identifier>
-          {firmware.osu && manager.formatHashName(firmware.osu.hash, deviceModelId, deviceInfo)}
+          {firmware.osu &&
+            manager
+              .formatHashName(firmware.osu.hash, deviceModelId, deviceInfo)
+              .map(hash => <span key={hash}>{hash}</span>)}
```

Comment:
Even if there is a low chance collision will happen, key should be index.

Reply:
used index in key